cpp/src/IceStorm/Service.cpp

```diff
@@ -103,9 +101,6 @@ ServiceI::start([[maybe_unused]] const string& serviceName, const CommunicatorPt
     auto topicAdapter = communicator->createObjectAdapter("IceStorm.TopicManager");
     auto publishAdapter = communicator->createObjectAdapter("IceStorm.Publish");
 
-    //
-    // We use the name of the service for the name of the database environment.
```

Comment:
This is not true, and LMDB environment don't have a "name", just a path. Also, unrelated to the code below.